app/lib/methods/helpers/sslPinning.ts

```diff
@@ -67,6 +67,7 @@ const RCSSLPinning = Platform.select({
 					certificate = persistCertificate(name, certificate.password);
 				}
 				UserPreferences.setMap(extractHostname(server), certificate);
+				SSLPinning?.setCertificate(server, certificate.path, certificate.password);
```

Comment:
What's going to happen to the current logged in users making use of pinning?
Are they going to be logged out?
Is this a breaking change?
If it is, should we refactor it to a better approach removing the need for a RN fork?

Reply:
It's not changing anything to the final user, there is no breaking change, I'm just creating a "copy" of the ssl pinning certificate on mmkv, currently our main app logic depends on a path to the certificate, for every place that a request is made we read this file, this can be changed in a future project. I had some issues with this logic based on the path, since for the Watch Connectivity the files can't be read if the app is not in foreground, so I created a code that will copy current certificates as raw data to this new key in our `key-value` storage, and new certificates will be also copied to this new key.